lib/Fhp/Segment/CAZ/GebuchteCamtUmsaetze.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Fhp\Segment\CAZ;
+
+use Fhp\Segment\BaseDeg;
+
+/**
+ * Needed for
+ * Issue: GetStatementOfAccountXML receiving multiple CAMT XML files
+ * @link: https://github.com/nemiah/phpFinTS/issues/370
+ *
+ * @link: https://github.com/nemiah/phpFinTS/pull/371
+ */
+class GebuchteCamtUmsaetze extends BaseDeg
+{
+    /** @var Bin[] @Max(99) */
+    public $gebuchteCamtUmsaetze;
+
+    /**
+     * Array of strings to store XML srings from the Bins structure.
+     * @var string[]
+     */
+    private $xml;
```

Comment:
This seems unused

Reply:
That's a local variable. To reference this `private $xml` from `getData()`, you would have to write `$this->xml` there.